prestodb/presto
# Add SQL Client page on Web UI

## Description
Leverage `presto-js-client` to run SQL queries and show results on the web UX. Add the limitation on
each SQL query to only return 100 records at most.

related to #21062

## Motivation and Context
Provide the capability to directly run simple SQL queries on the web UI and check results.

## Impact
Users accessing web UI can run SQL queries and check results on the new page. 

## Test Plan
1. check out the PR change
2. run a presto server using docker
3. run the web UI and proxy the API calls to the docker container which is running as a presto server
4. check the new page

## Contributor checklist


## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== NO RELEASE NOTE ==
```
